src/applications/ivc-champva/10-10D/tests/pages/applicantMedicareStatusPage.unit.spec.js

```diff
@@ -16,7 +16,7 @@ import mockData from '../fixtures/data/test-data.json';
 
 testComponentRender(
   'ApplicantMedicareStatusReviewPage',
-  <>{ApplicantMedicareStatusReviewPage()}</>,
+  <ApplicantMedicareStatusReviewPage data={mockData.data} title={() => ''} />,
```

Comment:
Fixing this unit test now that the custom page is slightly different underneath.